packages/components/src/card/card-divider/hook.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { cx } from 'emotion';
+
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useContextSystem } from '../../ui/context';
+import * as styles from '../styles';
+
+/**
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../../divider').DividerProps, 'hr'>} props
+ */
+export function useCardDivider( props ) {
+	const { className, ...otherProps } = useContextSystem(
+		props,
+		'CardDivider'
+	);
+
+	const classes = useMemo(
+		() =>
+			cx(
+				styles.Divider,
+				styles.borderColor,
+				'components-card__divider',
```

Comment:
Should we add a note that this classname (and the others like it) exist for legacy compatability reasons?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32566/commits/0b07472fcc8dc5eef5e8724f4c5ed9d66de30409